kbase-extension/static/kbase/js/widgets/appWidgets2/input/subdataInput.js

```diff
@@ -10,7 +10,7 @@ define([
     'common/ui',
     'common/props',
     'base/js/namespace',
-    '../subdataMethods/manager',
```

Comment:
rather than using one subdataMethods (I saw the comment about consolidating to one version, which is good) based in the input parameters directory place it one level above so that it is clear it is shared from that root; a dependency on a sister directory is a big confusing (things are in input or view directories specifically because they are or might exhibit behavior specific to being input parameters or view-only parameters.)